tests/rptest/tests/data_migrations_api_test.py

```diff
@@ -33,6 +33,7 @@
 
 MIGRATION_LOG_ALLOW_LIST = [
     'Error during log recovery: cloud_storage::missing_partition_exception',
+    'Requested data migration does not exist',
```

Comment:
this shouldn't be logged with an `error` severity as this is a result of user interacting with an API endpoint